src/index.spec.js

```diff
@@ -28,6 +28,14 @@ const data = deepFreeze({
       username: 'Administrator',
       password: 'pas:sw@ord'
     }
+  },
+  NFS: {
+    string: 'nfs://192.168.100.225:media/nfs',
```

Comment:
Ugly. The path is stored without a leading slash, and the / is added just in time by xo-server before mounting.